lib/house.rb

```diff
@@ -0,0 +1,58 @@
+class House
+  attr_reader :price, :address, :rooms
```

Comment:
Keep an eye on the interaction pattern - we wanted the `price` method to return an integer.  So, you could have removed this attr_reader, and just called your method `turn_price_to_int`, `price`.